ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/FilterParameterTest.java

```diff
@@ -224,6 +243,91 @@ public void emptyListAsSubTopicDecodesCorrectly() throws JsonProcessingException
             emptyList());
   }
 
+  @Test
+  public void emptyListOfTopicsDecodesCorrectly() throws java.io.IOException {
```

Comment:
nit: the naming is as good as the rest of Besu tests, but I always worry when I see "correctly" in tests, what is correctly? This is effectively going to be our requirements doc, so it needs to be explicit about the expected outcome.
eg. emptyListTopicStringDecodesToZeroLengthTopicList ... ok ... that's not great either. :/